src/runtime/runtime.cpp

```diff
@@ -252,10 +177,10 @@ main(int argc, char* argv[]) {
     std::unique_ptr<logging::logger_t> logger;
     std::unique_ptr<logging::log_t>    wrapper;
 
+    cocaine::logging::init_t logging(config->logging.loggers);
```

Comment:
It can not throw std::out_of_range and I use it to reset logger outside try block.